tests/BundledResources/BundledResources.csproj

```diff
@@ -48,6 +48,9 @@
   <ItemGroup>
     <BundleResource Include="..\monotouch-test\basn3p08.png">
       <Link>basn3p08.png</Link>
+    </BundleResource>
+        <BundleResource Include="basn3p08_with_loc.png">
```

Comment:
Looks like the indentation is wrong here.

Also I think the path should be "..\monotouch-test\basn3p08_with_loc.png"